salt/metalk8s/registry/installed.sls

```diff
@@ -47,6 +47,6 @@ Install OCI registry manifest:
     - defaults:
         registry_image: {{ registry_image }}
         registry_version: {{ registry_version }}
-        registry_ip: {{ pillar.registry_ip }}
+        registry_ip: {{ metalk8s.endpoints.registry.ip }}
```

Comment:
Hmh, no. This was wrong in the beginning as well.

This should be set to the `control_plane_ip` (grain) of the node on which the service is deployed.

`metalk8s.endpoints.*` is a *result* of a running service, not the *input* to configure a service.

Reply:
I will update it to control_plane_ip